ConsoleApp/Program.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Runtime;
+using System.IO;
+using System.Text;
+using static System.Console;
+const string filename = "data.csv";
+
+void ValidateIsNumber(string value)
+{
+    try
+    {
+        Int32.Parse(value);
+    }
+    catch
+    {
+        Console.WriteLine("Incorrect number...");
+        Console.ReadKey();
+        Environment.Exit(0);
+    }
+}
+
+void ValidateNoNumbers (string value)
```

Comment:
better to name is something like `ValidateNames` because `NoNumbers` is very wide)